src/script/team/TeamRepository.ts

```diff
@@ -156,6 +157,7 @@ export class TeamRepository extends TypedEventEmitter<Events> {
       try {
         await this.getTeam();
         await this.updateFeatureConfig();
+        this.emit('teamRefreshed');
```

Comment:
suggestion: maybe we could narrow down the scope of this event to just `featureConfigUpdated` and move it to the `updateFeatureConfig` function (this way we are sure this event will emit even if we just update the config and not the entire team). WDYT?

Reply:
`updateFeatureConfig` is called every time **any** of the feature is updated though, so we would refresh protocols too often in my opinion.

Reply:
Hummm I see. ok then, fine for now 👍